parquet-column/src/main/java/org/apache/parquet/example/data/simple/convert/GroupRecordConverter.java

```diff
@@ -46,7 +47,8 @@ public void end() {
 
   @Override
   public Group getCurrentRecord() {
-    return root.getCurrentRecord();
+    SimpleGroup simpleGroup = (SimpleGroup) root.getCurrentRecord();
+    return simpleGroup.isNull() ? null : simpleGroup;
```

Comment:
why do we want to skip a record where all values are null?
These are still valid records.